src/pages/blog/[...path].tsx

```diff
@@ -143,6 +142,48 @@ const BlogPage: FC<BlogPageProps> & {
     );
   }, [articles, blogMeta]);
 
+  const getJsonLd = (
+    blogMeta: Nullable<BlogArticleMeta>
+  ): Nullable<BlogArticleJsonLD | object> => {
+    if (blogMeta) {
+      return {
+        "@context": "https://schema.org",
+        "@type": "BlogPosting",
+        headline: blogMeta.title,
+        image: process.env.NEXT_PUBLIC_BASE_URL + blogMeta.themeImgSrc,
+        url: process.env.NEXT_PUBLIC_BASE_URL + "/blog/" + blogMeta.slug,
+        dateCreated: blogMeta.publishedOn,
+        dateModified: blogMeta.commit.lastEditedTime,
+        datePublished: blogMeta.publishedOn,
+        inLanguage: blogMeta.lang,
+        isFamilyFriendly: true,
+        keywords: [blogMeta.slug],
```

Comment:
We will have keywords frontmatter in our blogPost in the future. Let's leave this one as blank array this PR

```suggestion
        keywords: [],
```